src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,17 @@
 import React from 'react';
+
 import './MoviesList.scss';
+import { MovieCard } from '../MovieCard';
+import { Movie } from '../../types/Movie';
+
+export type Props = {
+  movies: Movie[];
+};
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+export const MoviesList: React.FC<Props> = ({ movies = [] }) => (
+  <>
+    {movies.map(movie => (
+      <MovieCard key={movie.imdbId} {...movie} />
```

Comment:
```suggestion
      <MovieCard key={movie.imdbId} movie={movie} />
```
It is better to pass the exact properties to avoid possible errors and make your code cleaner